src/axom/inlet/SphinxDocWriter.cpp

```diff
@@ -0,0 +1,95 @@
+#include "axom/inlet/SphinxDocWriter.hpp"
+#include <iostream>
+#include <assert.h>
+#include "axom/slic.hpp"
+
+namespace axom
+{
+namespace inlet
+{
+ 
+SphinxDocWriter::SphinxDocWriter(const std::string& fileName, axom::sidre::Group* root) {
+  SLIC_ASSERT_MSG(root != nullptr, "Sidre Group is null pointer");
+  m_sidreRootGroup = root;
+  m_fileName = fileName;
+}
+
+void SphinxDocWriter::writeDocuments(axom::sidre::Group* sidreGroup) {
+  if (sidreGroup->getName() == "") {
+    writeTitle("Untitled");

```

Comment:
Can we come up with a better title than untitled? maybe just command line options?